src/dal/Jalasoft.TeamUp.Resumes.DAL/ResumesRepository.cs

```diff
@@ -79,9 +80,21 @@ public class ResumesRepository : IResumesRepository
                 }
             };
 
+        private IConfigurationRoot config;
+
         public IEnumerable<Resume> GetResumes()
         {
-            return Resumes;
+            this.config = new ConfigurationBuilder()
+            .AddJsonFile("DBConfig.json", optional: false, reloadOnChange: true)
+            .Build();
+            string connectionString = this.config["appSettings:ConnectionString"];
+
+            IDapperWrapper wrapper = new DapperWrapper();
+
+            ResumeSQLRepository myRepository = new ResumeSQLRepository(wrapper, connectionString);
```

Comment:
the memory repository should not be calling the sql repository, memory repo is independent and it should not have any changes

Reply:
I fix this.